common/api/ui-framework.api.md

```diff
@@ -1910,7 +1910,7 @@ export const expandWidget: (base: {
         readonly home: {
             readonly widgetIndex: number;
             readonly widgetId: string | undefined;
-            readonly side: PanelSide;
+            readonly side: "top" | "left" | "right" | "bottom";

```

Comment:
Need to run "npm run clean" & "npm run build" before "npm run extract-api" in ui-framework.

Reply:
done.